app/models/art.rb

```diff
@@ -0,0 +1,10 @@
+class Art < ApplicationRecord
+  belongs_to :user
+  belongs_to :category
```

Comment:
The Category model doesn't exist yet so this shouldn't be in the code.
Adding it to this model will just be part of the Category model creation task

Reply:
I have deleted it.
Also I have changed has_one_attached :photo to: has_many_attached :photos

Reply:
I thought about that but will a piece of art really have multiple photos ? I guess it could
You'll need to update the params to permit :photos instead of :photo then